gitops-wg/README.MD

```diff
@@ -1,3 +1,89 @@
-# GitOps Working Group (Work In Progress)
+# GitOps Working Group
 
-Please check the [charter draft (WIP)](https://docs.google.com/document/d/11EZfvB2FFI837nMmArnyv-wizsIJvc-4_xdgfoUXF4o/edit) for the purpose & goals of the GitOps working group. Feel free to add yourself as interested parties and get involved in this effort.
\ No newline at end of file
+<!-- markdownlint-disable MD033 -->
+<p><img src="https://raw.githubusercontent.com/gitops-working-group/gitops-working-group/main/assets/logos/GWG/icon/color/gitopswg-icon-color.svg" alt="GitOps Working Group logo icon color" width="200"></p>
+
+The GitOps Working Group is a WG under the CNCF App Delivery SIG.
+
+The focus of the GitOps WG is to clearly define a vendor-neutral, principle-led meaning of GitOps. This will establish a foundation for interoperability between tools, conformance, and certification. Lasting programs, documents, and code are planned to live within the [OpenGitOps](https://github.com/open-gitops) project.
+
+See the [charter](charter.md) for additional info on the group's mission, scope, and alignment with other groups and projects.
+
+## Growing Adoption of GitOps
+
+The creation of the GitOps Working Group was driven by the accelerating adoption of GitOps tools and methodologies by users of services from Amazon, Codefresh, GitHub, Microsoft, Weaveworks, and hundreds of other leading global companies that are adopting GitOps.
+This, combined with the recommendation by the Cloud Native Computing Foundation (CNCF) [user community to adopt Flux](https://radar.cncf.io/2020-06-continuous-delivery), made it clear that GitOps is fast becoming the methodology of choice for operating modern cloud native infrastructure and applications.
+The CNCF user community reported that development, DevOps, and operations teams who adopt GitOps tooling and follow best practices experience improvements in productivity, stability, reliability, and security for their cloud native environments.
```

Comment:
Isn't stability and reliability duplicate?

Reply:
Good question. This was part of the original GitOps WG README, has never been contested. While I did watch Cheryl's YouTube and read the CD tech radar compiled results, I'm not sure where the details of this sentence were originally drawn from. I just left it as-is for now, with the idea that this README should be dramatically simplified, to only give info on the working group not the history of GitOps itself. This info was written before the OpenGitOps project was solidified, but it should probably now move to a supporting document in the OpenGitOps repo. Can that be a follow-up PR or would you rather this simplification happen now before merging this PR?

Reply:
Nm, it was drawn from the original WG announcement: https://www.weave.works/blog/announcing-gitops-working-group. I think it can be removed at this point, and the WG can discuss which parts and how to add as a supporting document in the OpenGitOps repo. Thank you

Reply:
Ok removing from here and tracking this question in https://github.com/open-gitops/project/discussions/23 so the info doesn't get lost.